tests/dummy/app/routes/nodes/detail/draft-registrations.js

```diff
@@ -0,0 +1,39 @@
+import Ember from 'ember';
+import permissions from 'ember-osf/const/permissions';
+
+export default Ember.Route.extend({
+    model() {
+        let node = this.modelFor('nodes.detail');
+        let drafts = node.get('draftRegistrations');
+        return Ember.RSVP.hash({
+            node: node,
+            drafts: drafts
+        });
+    },
+    actions: {
+        createDraft(schemaId) {
+            var node = this.modelFor(this.routeName).node;
+            if (node.get('currentUserPermissions').indexOf(permissions.ADMIN) !== -1) {
+                var draft = this.store.createRecord('draft-registration', {
+                    registrationSupplement: schemaId
+                });
+                node.get('draftRegistrations').pushObject(draft);
+                node.save();
+                node.one('didUpdate', this, function() {
```

Comment:
If it has been overridden, this should probably be fixed when we do the adapter cleanup.

(at one point we were altering what/whether `save` returned for relationship updates... in which case the solution is to carefully update our code to be consistent with Ember conventions)